### PR TITLE
use api logout

### PR DESCRIPTION
**Pull request by ghost**
## Description
use api logout
## Reference Requirements
- Task ID: TI-1

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

**Comment by at-sonle** [comment]
LGTM

File: app/Http/Controllers/Api/LoginController.php [hunk 1]
```diff
@@ -43,4 +43,17 @@ public function logout()
         $accessToken->revoke();
         return $this->successResponse(null, Response::HTTP_NO_CONTENT);
     }
+
+    /**
+     * Check access token api
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function checkAccessToken()
```
Comment (at-tuanphan) [inline app/Http/Controllers/Api/LoginController.php]: cái này viết trong `middleware` và sử dụng ở route chứ em?

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**